blackJack.js

```diff
@@ -0,0 +1,64 @@
+document.addEventListener("DOMContentLoaded",()=>{
+
+let button = document.createElement("button")
+
+const getGame = async () => {
+    try {
+        let res =  await axios.get("https://deckofcardsapi.com/api/deck/new/draw/?count=1");
```

Comment:
Here you are getting a new deck and drawing 1 card. 

You can change the values and the url to get different responses.